packages/base/card-api.gts

```diff
@@ -1905,6 +1916,17 @@ export function unsubscribeFromChanges(
     return;
   }
   changeSubscribers.delete(subscriber);
+
+  let fields = getFields(fieldOrCard, {
+    usedFieldsOnly: true,
+    includeComputeds: false,
+  });
+  Object.keys(fields).forEach((fieldName) => {
+    let value = peekAtField(fieldOrCard, fieldName);
+    if (isCompoundField(value)) {
```

Comment:
same comments as the subscribe--use `isCardOrField()`, and also recursively unsubscribe as field can have card that have fields that are cards that can have fields that are cards, etc...